README.md

```diff
@@ -64,6 +64,36 @@ Or add to your Cargo.toml:
 markdown2pdf = "0.1.9"
 ```
 
+### Feature Flags
+
+The library provides optional feature flags to control dependencies:
+
+- **Default (no features)**: Core PDF generation from files and strings. No network dependencies.
+- **`fetch`**: Enables URL fetching support (requires one of the TLS features below).
+- **`native-tls`**: Enables URL fetching with native TLS/OpenSSL (recommended for most users).
+- **`rustls-tls`**: Enables URL fetching with pure-Rust TLS implementation (useful for static linking or avoiding OpenSSL).
+
+```toml
+# Minimal installation (no network dependencies)
+markdown2pdf = { version = "0.1.9", default-features = false }
```

Comment:
I think you don't need the `default-features = false` to run without networking. That would only be needed if `fetch` was enabled by default. But since your new default is `[]`, it's fine.

Reply:
Resolved, thanks!